translations/english.json

```diff
@@ -192,35 +192,35 @@
 
         "Paired Statue": "Paired Statue",
-        "Solanum just happened to pass right by the unpaired statue inside the Ash Twin Project. Now she's stuck in the time loop too.": "Solanum happened to pass by the unpaired statue inside the Ash Twin Project. Now she's stuck in the time loop too.",
+        "Solanum just happened to pass right by the unpaired statue inside the Ash Twin Project. Now she's stuck in the time loop too.": "Solanum happened to pass by the unpaired statue inside the Ash Twin Project. Now she's aware of the time loop too.",
         // "Now that Solanum knows how time loop reset works, we will try reach the Vessel.": "Now that Solanum knows how time loop reset works, we will try reach the Vessel.",
-        "Last cycle, Solanum was not sure how the time loop worked. This cycle it seems like she understood everything and now wants to go to the Vessel.": "Last cycle, Solanum was not sure how the time loop worked. This cycle it seems like she understood everything and now wants to go to the Vessel.",
+        "Last cycle, Solanum was not sure how the time loop worked. This cycle it seems like she understood everything and now wants to go to the Vessel.": "Last cycle, Solanum was not sure how the time loop worked. This cycle it seems as if she understands everything, and wants to go to the Vessel.",
 
 
         // Ernesto Quest        
         "Where is Ernesto?": "Where is Ernesto?",
         "Seriously, how could an anglerfish just disappear?": "Seriously, how could an anglerfish just disappear?",
-        "Hornfels said that anglerfish «Ernesto» disappeared from exhibit.": "Hornfels said that anglerfish «Ernesto» disappeared from exhibit.",
+        "Hornfels said that anglerfish «Ernesto» disappeared from exhibit.": "Hornfels said that the anglerfish \"Ernesto\" disappeared from its exhibit.",
         "Ernesto found": "Ernesto found",
         "I found Ernesto": "I found Ernesto",
-        "I found Slate with Ernesto. He's cooking him on campfire.": "I found Slate with Ernesto. He's cooking him on campfire.",
-        "Slate is just fed up with marshmallow, that's why he decided to cook Ernesto. He said it will be ready in 30 minutes. Intervention isn't required because the loop will reset before that happens.": "Slate is just fed up with marshmallow, that's why he decided to cook Ernesto. He said it will be ready in 30 minutes. Intervention isn't required because the loop will reset before that happens."
+        "I found Slate with Ernesto. He's cooking him on campfire.": "I found Slate with Ernesto. They're cooking it on a campfire.",
```

Comment:
They're? Slate is cooking Ernesto, no?

Reply:
Slate is a they, the Hearthians have a single gender and use they as a pronoun in English.

Reply:
> Slate is a they, the Hearthians have a single gender and use they as a pronoun in English.

what? is there anything in game about it?